video/out/vo_dmabuf_wayland.c

```diff
@@ -691,6 +700,11 @@ static int reconfig(struct vo *vo, struct mp_image *img)
     if (!vo_wayland_reconfig(vo))
         return VO_ERROR;
 
+    mp_mutex_lock(&vo->params_mutex);
+    p->target_params = img->params;
```

Comment:
I think we should sanitize it a little, because dovi and hdr10+ wouldn't be supported anyway.

```suggestion
    p->target_params = img->params;
    // Restore fallback layer parameters if available.
    mp_image_params_restore_dovi_mapping(&p->target_params);
    // Strip metadata that are not understood anyway.
    const struct pl_hdr_metadata *hdr = &p->color.hdr;
    hdr->scene_max[0] = hdr->scene_max[1] = hdr->scene_max[2] = 0;
    hdr->scene_avg = hdr->max_pq_y = hdr->avg_pq_y = 0;
```

For DV it allows to use HDR10/HLG fallback. And the rest is mostly cosmetic to make it clear that we don't send those metadata further.

Reply:
`mp_image_params_restore_dovi_mapping` makes sense. Is the other stuff related to the stats just to make sure we don't report nonsense? I didn't look yet.

Reply:
Indeed, the other stuff is to not report nonsense on output.